tests/examplefiles/lean/test.lean

```diff
@@ -207,3 +207,7 @@ let ⟨m, hm⟩ := @zorn α (≤) h (assume a b c, le_trans) in
 ⟨m, assume a ha, le_antisymm (hm a ha) ha⟩
 
 end zorn
+
+-- other bits of tricky syntax
+@[to_additive "See note [foo]"]
+lemma mul_one : sorry := sorry
```

Comment:
Note that github is currently stuck on https://github.com/leanprover/vscode-lean/releases/tag/v0.16.26, and the fix didn't make it in until v0.16.28